src/index.js

```diff
@@ -396,28 +397,11 @@ walletRouter.post('/simple-send-tx',
     },
     token: {
       in: ['body'],
-      isObject: true,
+      isString: true,
       optional: true,
-      custom: {
```

Comment:
I like this change, but we are changing the API of the `/simple-send-tx`. It means all integrations using it must adapt their code. Should we create a new version of the API and mark this one as deprecated?